morango/syncsession.py

```diff
@@ -55,6 +61,14 @@ def _get_client_ip_for_server(server_host, server_port):
     return IP
 
 
+# Used from https://github.com/django/django/blob/1.11.20/django/utils/text.py#L295
```

Comment:
Might be clearer as "Borrowed from..." -- I was trying to figure out why Django would be importing from Morango! :)